scripts/read_audio_performance.sh

```diff
@@ -0,0 +1,17 @@
+# generate test files
+# ffmpeg -f lavfi -i "sine=frequency=1000:duration=3600" -c:a pcm_s16le -ar 8000 data/tests/test_1h.wav
+# ffmpeg -f lavfi -i "sine=frequency=1000:duration=60" -c:a pcm_s16le -ar 8000 data/tests/test_1m.wav
+# ffmpeg -f lavfi -i "sine=frequency=1000:duration=5" -c:a pcm_s16le -ar 8000 data/tests/test_5s.wav
+
+# wav
+python audio.py timeit --audio-path data/tests/test_5s.wav --mono --audio-backend soundfile --numbers 100
```

Comment:
Please use a loop instead. Also, no need to specify `--numbers`. We probably won't change the default value anyway.

Reply:
done